ui/server/src/main/scala/pl/touk/nussknacker/ui/process/ProcessService.scala

```diff
@@ -0,0 +1,115 @@
+package pl.touk.nussknacker.ui.process
```

Comment:
Can we add description what should go here?
I wonder, maybe it would be better to merge ProcessService with ManagementService? The boundaries are not so clear  for me. If there is good reason for separation, let's write it in some comment?